src/main/java/edu/wpi/grip/ui/PaletteView.java

```diff
@@ -49,6 +105,12 @@ public void clearOperations() {
 
     @Subscribe
     public void onOperationAdded(OperationAddedEvent event) {
-        this.operations.getChildren().add(new OperationView(this.eventBus, event.getOperation()));
+        OperationView view = new OperationView(this.eventBus, event.getOperation());
+        this.operations.getChildren().add(view);
+        FXCollections.sort(this.operations.getChildren(),
```

Comment:
Yeah, instead of adding the search feature.  This change also adds a search feature.

Either way, alphabetizing every operation makes it difficult to browse them because the related ones are all scrambled around.  If you think the generated ones should be alphabetized, maybe we can just make the generator code generate them in alphabetical order.